src/entrypoints/core.ts

```diff
@@ -110,24 +109,3 @@ window.hassConnection.then(({ conn }) => {
     llWindow.llResProm = fetchResources(conn);
   }
 });
-
-window.addEventListener("error", (e) => {
```

Comment:
Why remove this?

Reply:
I see now that you moved this. We should not. 

This needs to be loaded as soon as possible or else we're going to miss events. 

If you prefer to move some logic to a later place, at least start listening for events and store them in a list for later processing.

Reply:
It only did something when hass was available, I made sure it is still added before hass is initialized

Reply:
but what if `core.ts` is having errors? In that case we are not capturing those errors. I think that we should at least capture all the errors until HASS has a connection established.

Reply:
OK, let's move this to another PR :-)